turbo.json

```diff
@@ -160,7 +160,7 @@
       "dependsOn": ["//#install:modules", "^build"]
     },
     "test:types": {
-      "dependsOn": ["//#install:modules", "^build", "build:idl"]
+      "dependsOn": ["//#install:modules", "^build", "build:idl", "build"]
```

Comment:
You might want to scope this only to the sessions-demo package, since most packages don't need to be built to type check and enforcing this will slow things down